core/frontend/src/test/tile/TileAdmin.test.ts

```diff
@@ -130,6 +130,10 @@ describe("TileAdmin", () => {
         resolve(undefined);
       }
 
+      public get requestChannel() {
```

Comment:
may be too late to change, but i complained about this elsewhere: variables/methods with `request` prefix are confusing because "request" is usually a verb. Can this just be `channel`?

Reply:
It's a property, not a method, so it's inherently a noun.

Reply:
I know, that's why it's so confusing for a property to start with a verb.

Reply:
Renamed because I hate arguing about names.